src/main/java/edu/cornell/kfs/concur/ConcurParameterConstants.java

```diff
@@ -10,5 +10,14 @@
     public static final String CONCUR_CUSTOMER_PROFILE_SUB_UNIT = "CONCUR_CUSTOMER_PROFILE_SUB_UNIT";
     public static final String DEFAULT_TRAVEL_REQUEST_OBJECT_CODE = "DEFAULT_TRAVEL_REQUEST_OBJECT_CODE";
     public static final String CONCUR_PENDING_CLIENT_OBJECT_CODE_OVERRIDE = "CONCUR_PENDING_CLIENT_OBJECT_CODE_OVERRIDE";
-
+    public static final String CONCUR_SAE_COLLECTOR_DOCUMENT_TYPE = "CONCUR_SAE_COLLECTOR_DOCUMENT_TYPE";
+    public static final String CONCUR_SAE_COLLECTOR_SYSTEM_ORIGINATION_CODE = "CONCUR_SAE_COLLECTOR_SYSTEM_ORIGINATION_CODE";
+    public static final String CONCUR_SAE_COLLECTOR_CHART_CODE = "CONCUR_SAE_COLLECTOR_CHART_CODE";
+    public static final String CONCUR_SAE_COLLECTOR_HIGHEST_LEVEL_ORG_CODE = "CONCUR_SAE_COLLECTOR_HIGHEST_LEVEL_ORG_CODE";
+    public static final String CONCUR_SAE_COLLECTOR_DEPARTMENT_NAME = "CONCUR_SAE_COLLECTOR_DEPARTMENT_NAME";
+    public static final String CONCUR_SAE_COLLECTOR_CAMPUS_CODE = "CONCUR_SAE_COLLECTOR_CAMPUS_CODE";
+    public static final String CONCUR_SAE_COLLECTOR_CAMPUS_ADDRESS = "CONCUR_SAE_COLLECTOR_CAMPUS_ADDRESS";
+    public static final String CONCUR_SAE_COLLECTOR_NOTIFICATION_CONTACT_EMAIL = "CONCUR_SAE_COLLECTOR_NOTIFICATION_CONTACT_EMAIL";
```

Comment:
I was planning to use parameters for these values, since the specs already mention which values should be used. I think only a few of these are in the customer profile, and the customer profile is in PDP so I'm not sure if it would be that useful for this GL processing.